wdl/CramToBam.wdl

```diff
@@ -22,6 +27,17 @@ workflow CramToBam {
       email: "tbrookin@broadinstitute.org"
   }
 
+  if (defined(service_account_json)) {
+    call util.LocalizeCloudFileWithCredentials as LocalizeCram {
+      input:
+        cloud_file_path = cram_file,
+        service_account_json = select_first([service_account_json]),
+        disk_size = select_first([localize_cram_disk_size, 50]),
+        cloud_sdk_docker = select_first([cloud_sdk_docker]),
+        runtime_attr_override = runtime_attr_localize_cram
+    }
+  }
+
   if (requester_pays) {
```

Comment:
```suggestion
  if (requester_pays && !defined(service_account_json)) {
```

Reply:
If the cram is in a requester pays bucket, I think it would fail before this point because there is no billing project provided to the `gsutil cp` command in the localization task